pkg/leakutil/leak_helper.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package leakutil
+
+import (
+	"testing"
+
+	"go.uber.org/goleak"
+)
+
+func SetUpLeakTest(m *testing.M, options ...goleak.Option) {
```

Comment:
I'm wondering why ci lint doesn't report `exported function SetUpLeakTest should have comment or be unexported`

Reply:
add comment for exported function SetUpLeakTest ---->done

Reply:
The comment style is `SetUpLeakTest ...`, should be started with function name
ref: https://golang.org/doc/effective_go#commentary
In old days the lint could check this, seems it does not work now.

Reply:
Seems the behavior changed after https://github.com/pingcap/ticdc/pull/2756 @hi-rustin would you please take a look?